### PR TITLE
Dfr 2340 avoid additional hearing document duplicates

### PR DESCRIPTION
**Pull request by jamiephillipshmcts**
https://tools.hmcts.net/jira/browse/DFR-2340

Prevent the additonalhearingdocument.pdf from being attached to all hearings after the one in which it should be attached in the Send Order (contested) event

### CODE REVIEW

**Review by ruban72 — DISMISSED** [review]
Looks like the shouldAddDocumentToOrderColl method is duplicated across 3 classes. Can we have a single reusable method here

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -210,4 +210,18 @@
         <packageUrl regex="true">^pkg:maven/org\.springframework\.plugin/spring\-plugin\-metadata@.*$</packageUrl>
         <cve>CVE-2023-34047</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-plugin-metadata-1.2.0.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\.plugin/spring\-plugin\-metadata@.*$</packageUrl>
+        <cpe>cpe:/a:spring:spring</cpe>
+    </suppress>
```
Comment (rishikrsharma) [inline config/owasp/suppressions.xml:219]: remove this - sync with master - its duplicating here

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -210,4 +210,11 @@
         <packageUrl regex="true">^pkg:maven/org\.springframework\.plugin/spring\-plugin\-metadata@.*$</packageUrl>
         <cve>CVE-2023-34047</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
```
Comment (rishikrsharma) [inline config/owasp/suppressions.xml:214]: its already suppressed - why again?

Comment (rishikrsharma) [reply config/owasp/suppressions.xml:214]: @jamiephillipshmcts

**Review by rishikrsharma — APPROVED** [review]

**Pull request merged**